Projects/Forever/Sources/Components/InivtationTable.swift

```diff
@@ -57,20 +57,21 @@ struct InvitationTable: View {
             }
         ) { foreverData in
             if let foreverData {
-                if !foreverData.invitations.isEmpty {
-                    hSection {
+                hSection {
+                    if !foreverData.referrals.isEmpty {
                         hRow {
                             hText(L10n.foreverReferralListLabel)
                         }
-                        ForEach(foreverData.invitations, id: \.hashValue) { row in
+                        ForEach(foreverData.referrals, id: \.hashValue) { row in
                             InvitationRow(row: row)
                         }
-                        getOtherDiscountsRow(foreverData)
-                        getTotalRow(foreverData)
                     }
-                    .sectionContainerStyle(.transparent)
-                    .padding(.horizontal, -16)
+                    getOtherDiscountsRow(foreverData)
```

Comment:
please check this function getOtherDiscountsRow()
 if let otherDiscounts = foreverData.otherDiscounts { <- it looks odd if we have -0 for "your other discounts"

}

Reply:
Agree. Will change this.